.gitignore

```diff
@@ -3,6 +3,7 @@
 *.swo
 *.pyc
 node_modules
+.cfmeta
```

Comment:
As we just found out, this file can't be ignored in order for the notifications to work properly.

Also, the `.cfignore` file is syminked to `.gitignore` as a heads up.